code/game/gamemodes/cult/cult_items.dm

```diff
@@ -72,6 +72,173 @@
 		if(is_servant_of_ratvar(C) && C.reagents)
 			C.reagents.add_reagent("heparin", 1)
 
+/obj/item/twohanded/required/cult_bastard
+	name = "bloody bastard sword"
+	desc = "An enormous sword used by Nar-Sien cultists to rapidly harvest the souls of non-believers."
+	w_class = WEIGHT_CLASS_HUGE
+	block_chance = 50
+	throwforce = 20
+	force = 40
+	armour_penetration = 30
+	throw_speed = 1
+	throw_range = 3
+	sharpness = IS_SHARP
+	light_color = "#ff0000"
+	attack_verb = list("cleaved", "slashed", "torn", "hacked", "ripped", "diced", "carved")
+	icon_state = "cultbastard"
+	item_state = "cultbastard"
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	lefthand_file = 'icons/mob/inhands/64x64_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/64x64_righthand.dmi'
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+	actions_types = list()
+	flags_2 = SLOWS_WHILE_IN_HAND_2
+	var/datum/action/innate/spin2win/linked_action
+	var/spinning = FALSE
+	var/dash_toggled = FALSE
+	var/charges = 1
+	var/charge_rate = 250
+	var/list/shards = list()
+
+/obj/item/twohanded/required/cult_bastard/Initialize()
+	. = ..()
+	set_light(4)
+
+/obj/item/twohanded/required/cult_bastard/can_be_pulled(user)
+	return FALSE
+
+/obj/item/twohanded/required/cult_bastard/attack_self(mob/user)
+	dash_toggled = !dash_toggled
+	if(dash_toggled)
+		to_chat(loc, "<span class='notice'>You raise the [src] and prepare to jaunt with it.</span>")
+	else
+		to_chat(loc, "<span class='notice'>You lower the [src] and prepare to swing it normally.</span>")
+
+/obj/item/twohanded/required/cult_bastard/pickup(mob/living/user)
```

Comment:
wait so if security somehow downs someone using this they can't even dispose of the damn weapon without using gravity guns/gravipults/atmos because it's undraggable and not able to be picked up?
the fuck???

Reply:
I actually changed that recently. I couldn't decide whether it was better to make holding the sword a 100% guaranteed "antag flag" or let anyone hold it so that it could be safely stored. 

I leaned toward the former because this weapon felt like massive greytide bait where the moment it hits the floor any bystanding will run off with it just for show. I'll probably compromise by making it easier to exorcise so security can pick up some bibles from the library and purge the sword. 

Plus if you exorcise it you get a holy claymore (null rod version) that makes you immune to talismans. There's a high payoff for neutralizing one of these weapons.

Reply:
>There's a high payoff for neutralizing one of these weapons.

More reason to make it limited to cult leader
You just KNOW someone will fuck up and give security a extra null rod